compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangParserListener.java

```diff
@@ -1444,6 +1444,60 @@ public void exitListConstructorExpr(BallerinaParser.ListConstructorExprContext c
         this.pkgBuilder.addListConstructorExpression(getCurrentPos(ctx), getWS(ctx), argsAvailable);
     }
 
+    @Override
+    public void enterLetExpression(BallerinaParser.LetExpressionContext ctx) {
+        if (isInErrorState) {
+            return;
+        }
+
+        this.pkgBuilder.startLetVarList();
+    }
+
+    @Override
```

Comment:
Do we need this method, I think `enterLetExpression` cover this parser event

Reply:
Fixed